extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -953,10 +955,12 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {!Promise} awaiting load event for ad frame
    * @private
    */
-  iframeRenderHelper_(iframe) {
+  iframeRenderHelper_(iframe, sentinel) {
     // TODO(keithwrightbos): noContentCallback?
     this.xOriginIframeHandler_ = new AMP.AmpAdXOriginIframeHandler(this);
-    return this.xOriginIframeHandler_.init(iframe, /* opt_isA4A */ true);
+    this.rendered_ = true;
```

Comment:
hm good question. I don't recall ever writing that line, I wonder if that is a holdover from some merge-conflict-resolution at some point. will remove.